.github/workflows/build-opcua-monitoring-broker-container.yml

```diff
@@ -42,11 +42,12 @@ jobs:
       uses: docker/setup-buildx-action@v3
 
     - name: Login to GitHub Container Registry
```

Comment:
I think the issue may be that these creds are not persisting to the push step. Can we move the login to the build step and keep using the `secrets.GITHUB_TOKEN` approach

Reply:
The credentials are persisting to the push step.
The GitHub token issue comes from the fact that you need to give the action permission to push these packages individually in the package parameters as org admin.
This would prevent us from renaming or creating new packages using this method.